package-lock.json

```diff
@@ -23386,17 +23389,67 @@
 			}
 		},
 		"node_modules/core-js-compat": {
-			"version": "3.31.0",
-			"resolved": "https://registry.npmjs.org/core-js-compat/-/core-js-compat-3.31.0.tgz",
-			"integrity": "sha512-hM7YCu1cU6Opx7MXNu0NuumM0ezNeAeRKadixyiQELWY3vT3De9S4J5ZBMraWV2vZnrE1Cirl0GtFtDtMUXzPw==",
+			"version": "3.35.1",
+			"resolved": "https://registry.npmjs.org/core-js-compat/-/core-js-compat-3.35.1.tgz",
+			"integrity": "sha512-sftHa5qUJY3rs9Zht1WEnmkvXputCyDBczPnr7QDgL8n3qrF3CMXY4VPSYtOLLiOUJcah2WNXREd48iOl6mQIw==",
 			"dependencies": {
-				"browserslist": "^4.21.5"
+				"browserslist": "^4.22.2"
 			},
 			"funding": {
 				"type": "opencollective",
 				"url": "https://opencollective.com/core-js"
 			}
 		},
+		"node_modules/core-js-compat/node_modules/browserslist": {
```

Comment:
This is bad, the `browserslist` and `caniuse-lite` should be deduped at root level. There are here because `core-js-compat` requires newer versions than are pinned in the root `package.json`. Let's fix that in a followup PR.

Reply:
@sirreal, updated `caniuse-lite` today. Maybe, it only requires a rebase?

Reply:
Looking into it now